src/main/charts/bamboo/values.yaml

```diff
@@ -341,9 +350,13 @@ ingress:
 #
 bamboo:
 
-  # -- Supply the base URL for the installation.
+  # -- To skip the setup wizard post deployment set this property to 'true' and
+  # ensure values for all 'REQUIRED' and 'UNATTENDED' stanzas 
+  # (see banner of this file) have been supplied.
+  #
+  # For release 1.0.0 this value is by default set to 'true'
   #
-  baseUrl:
+  unattendedSetup: true
```

Comment:
What is the play after we will support a partial unattended setup? Will this be `true` and then just customer provides less properties?

What will happen if this is set to `false`? Just from curioisity.

Reply:
So:

- `Full` unattended setup this should be `true`
- `Partial` unattended setup this should be `true`
- Attended setup this should be `false`

Im not a fan of this, and I'd like the fix being addressed by the Bamboo team to negate the need for this.